### DISCUSSION TITLE
HDarea高清视界 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#30] [模板讨论]
HDarea高清视界 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by 18690819111** [comment]
失效了，应该是网站套cf了

**Reply by wjf0214** [reply]
PT网站我不维护了，没账号，太麻烦。这还是早期别人要的时候做的。不过我看了下，貌似没有cf盾。可以试试看更新下cookie试试。可能是过期了吧。

**Reply by 18690819111** [reply]
谢谢大佬提示
